redbot/cogs/mod/kickban.py

```diff
@@ -173,6 +173,18 @@ class KickBanMixin(MixinMeta):
             await ctx.send(_("I cannot do that due to discord hierarchy rules"))
             return
         audit_reason = get_audit_reason(author, reason)
+        if reason == None:
```

Comment:
`None` is singleton so you should use identity check
```suggestion
        if reason is None:
```